### PR TITLE
Add com.voxdsp.AstroImpact

### PR DESCRIPTION
**Pull request by BeverlyCode**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by bbhtt** [comment]
Please try to build at least once locally instead of here

File: com.voxdsp.AstroImpact.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - --share=ipc
+  - --device=dri
+  - --share=network
+modules:
+  - name: glfw
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_SHARED_LIBS=ON
+      - -DGLFW_BUILD_EXAMPLES=OFF,
```
Comment (bbhtt) [inline com.voxdsp.AstroImpact.yaml:21]:
```suggestion
      - -DGLFW_BUILD_EXAMPLES=OFF
```

**Comment by bbhtt** [comment]
It would've been apparent if you at least attempted to build once locally, instead you kept pushing...

It's now failing here

```
Renaming astroimpact.desktop to com.voxdsp.AstroImpact.desktop
Error: Error on line 10 char 278: Element “p” was closed, but the currently open element is “br”
```

**Comment by bbhtt** [comment]
> Yeah I should workout how to do the whole `bot, build` thing locally sometime.

https://docs.flatpak.org/en/latest/first-build.html

File: com.voxdsp.AstroImpact.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+rename-icon: astroimpact
+rename-desktop-file: astroimpact.desktop
+rename-appdata-file: astroimpact.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --filesystem=home
```
Comment (hfiguiere) [inline com.voxdsp.AstroImpact.yaml:15]: why do you need home?

**Pull request closed without merging**